src/app/pages/server-details/server-details.component.html

```diff
@@ -630,6 +684,42 @@ <h3 class="text-2xl" >Similar Servers </h3>
   </ul>
 </div>
 
+<div id="static_web_options" class="z-10 hidden bg-secondary rounded-lg shadow w-44 border border-emerald-400 border-solid">
+  <ul class="py-2 px-1 text-sm text-gray-700" aria-labelledby="dropdownDefaultButton">
+    <li *ngFor="let item of staticWebOptions" class="flex py-1 gap-2 items-center">
+        <div
+          class="text-emerald-400 hover:text-emerald-500 cursor-pointer font-bold ml-2"
+          (click)="selectStaticWebOption(item)">
+          {{item.name}}
+        </div>
+    </li>
+  </ul>
+</div>
+
+<div id="static_web_options" class="z-10 hidden bg-secondary rounded-lg shadow w-44 border border-emerald-400 border-solid">
+  <ul class="py-2 px-1 text-sm text-gray-700" aria-labelledby="dropdownDefaultButton">
+    <li *ngFor="let item of redisOptions" class="flex py-1 gap-2 items-center">
+        <div
+          class="text-emerald-400 hover:text-emerald-500 cursor-pointer font-bold ml-2"
+          (click)="selectedRedisOption(item)">
+          {{item.name}}
+        </div>
+    </li>
+  </ul>
+</div>
+
```

Comment:
what's this? called static_web_options, populated by the redis options?
seem to be a copy/paste mutant of the above and below divs?